src/stamp.rs

```diff
@@ -1,10 +1,13 @@
 use crate::{ArticleId, ArticleIdError, CategoryId};
-use jiff::civil::Date;
-use jiff::fmt::strtime::format as jiff_format;
-use jiff::fmt::strtime::parse as jiff_parse;
-use jiff::Error as JiffError;
-use std::error::Error;
-use std::fmt::{Display, Formatter, Result as FmtResult};
+use jiff::{
+	civil::Date,
+	fmt::strtime::{format as jiff_format, parse as jiff_parse},
+	Error as JiffError,
+};
+use std::{
+	error::Error,
+	fmt::{Display, Formatter, Result as FmtResult},
+};
```

Comment:
Could you revert these changes? I personally prefer the previous style of imports (semi-flat) for diffing + readability, where its nested at the very bottom level instead of the top.